salt/roster/dir.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+'''
+Read in the rosters in a directory using the renderer system.
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import os
+from fnmatch import fnmatch
+import re
+import copy
+import logging
+
+import salt.loader
+import salt.config
+from salt.template import compile_template
+
+from salt.roster.flat import HAS_RANGE, _convert_range_to_list
+
+# Import Salt libs
+
+log = logging.getLogger(__name__)
+
+
+def targets(tgt, tgt_type='glob', **kwargs):
+    '''
+    Return the targets from the directory of flat yaml files,
+    checks opts for location.
+    '''
+    roster_dir = __opts__.get['roster_dir']
```

Comment:
`__opts__.get('roster_dir', '/path/to/default')`

Reply:
wow, how did that slip by?